CONTRIBUTORS.md

```diff
@@ -87,3 +87,5 @@ This is a list of people who have contributed code to the [YugabyteDB](https://g
 * [eliahburns](https://github.com/eliahburns)
 * [bhavin192](https://github.com/bhavin192)
 * [graffido](https://github.com/graffido)
+* [svanellewee](https://github.com/svanellewee)
+* [nvulane](https://github.com/nvulane)
```

Comment:
Hi @svanellewee ,

Would be good if @nvulane can send a separate PR with this entry. This way, the listing on github contributors page for yugabyte would also be accurate: https://github.com/yugabyte/yugabyte-db/graphs/contributors

Could you please modify the PR to just add your entry? Thanks!

Reply:
Sure thing, I'll ping him.

Reply:
Oh! I guess it makes  sense, means that he is also signing his contributor agreement.